modern/org/commcare/modern/database/TableBuilder.java

```diff
@@ -180,7 +180,7 @@ public String getTableCreateString() {
 
     //sqlite doesn't like dashes
     public static String scrubName(String input) {
-        return input.replace("-", "_");
+        return input.replace("-", "_dash_").replace(".", "_dot_").replace("+", "_plus_");
```

Comment:
I would've expected android to crash if it did have usernames like that. though it seems plausible that it would have come across `.` in usernames. Do you know if Android's sqlite version is just able to handle that? Or is there some other escaping process